lbrynet/lbrynet_daemon/auth/server.py

```diff
@@ -90,7 +97,16 @@ def __init__(self, use_authentication=settings.use_auth_http):
     def setup(self):
         return NotImplementedError()
 
+    def _render_error(self, request, failure, version=jsonrpclib.VERSION_1, response_code=FAILURE):
+        fault = jsonrpclib.dumps(JSONRPCException(Failure(failure), response_code), version=version)
```

Comment:
nit: long line

Also, why is `Failure(failure)` necessary as compared to only `failure`